src/pages/evm/wallet/SendPage.vue

```diff
@@ -1,21 +1,577 @@
+                        <template v-slot:selected>
+                            <span>{{ token?.symbol }}</span>
+                        </template>
+
+                        <template v-slot:option="scope">
+                            <q-item class="c-send-page__selector-op" v-bind="scope.itemProps">
+                                <div class="c-send-page__selector-op-avatar">
+                                    <img class="c-send-page__selector-op-icon" :src="scope.opt.logoURI" alt="Token Logo">
+                                </div>
+                                <div>
+                                    <q-item-label class="c-send-page__selector-op-name">{{ scope.opt.name }}</q-item-label>
+                                    <q-item-label class="c-send-page__selector-op-balance" caption>
+                                        {{ prettyPrintBalance(scope.opt.balance, userStore.fiatLocale, isMobile, scope.opt.symbol) }}
+                                    </q-item-label>
+                                </div>
+                            </q-item>
+                        </template>
+                    </q-select>
+                    <div
+                        :class="{
+                            'c-send-page__view-contract': true,
+                            'c-send-page__view-contract--hidden': !showContractLink,
+                        }"
+                        @click="viewTokenContract"
+                    >
+                        <span class="c-send-page__view-contract-text">{{ $t('evm_wallet.view_contract') }}</span>
+                        <q-space v-if="!isMobile"/>
+                        <q-icon size="xs" name="launch" class="c-send-page__view-contract-min-icon" />
+                    </div>
+                </div>
+                <!-- Amount input -->
+                <div class="c-send-page__amount-col col">
+                    <div v-if="isMobile" class="row c-send-page__available">
+                        <q-space/>
+                        <div class="col-auto">
+                            <span class="c-send-page__amount-available" @click="setAllBalance">
+                                {{ $t('evm_wallet.amount_available', { amount:available }) }}
+                            </span>
+                        </div>
+                    </div>
+                    <q-input
+                        v-model="amount"
+                        outlined
+                        class="c-send-page__amount-input"
+                        :label="$t('evm_wallet.amount')"
+                        :rules="[
+                            val => !!val || $t('evm_wallet.amount_required'),
+                            val => isPrecisionCorrect || $t('evm_wallet.invalid_amount_precision', { precision: useFiat ? 2 : (token?.decimals ?? 0) })
+                        ]"
+                        type="number"
+                        step="0.01"
+                        pattern="[0-9]*\.?[0-9]+"
```

Comment:
this is fine for now but we need the currency input to be a lot more robust and reusable. i have created a ticket for this, which i will implement :angel: 
https://github.com/telosnetwork/telos-wallet/issues/262